### PR TITLE
Feat/add author pr commits

### PR DESCRIPTION
**Pull request by jaorr95**
## Description

Add `author` and `createdAt` information for commits related with Pull Requests. 

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: destinations/airbyte-faros-destination/src/converters/github/pull_request_commits.ts [hunk 1]
```diff
@@ -24,9 +25,13 @@ export class PullRequestCommits extends GitHubConverter {
 
     if (!repository) return [];
 
+    const author = prCommit.author ? {uid: prCommit.author.login, source} : null;
+
     const commit = {
       uid: prCommit.sha,
       sha: prCommit.sha,
+      author,
```
Comment (tovbinm) [inline destinations/airbyte-faros-destination/src/converters/github/pull_request_commits.ts:33]: Author information is already present on the `vcs_Commit` model and being written here - https://github.com/jaorr95/airbyte-connectors/blob/feat/add-author-pr-commits/destinations/airbyte-faros-destination/src/converters/github/commits.ts#L35

I.e. when querying Faros GraphQL you can access it by going `vcs_Commit` -> `author` (same applies to `createdAt` timestamp.

Comment (jaorr95) [reply destinations/airbyte-faros-destination/src/converters/github/pull_request_commits.ts:33]: Yes, but that information is only filled for the main branches of the repo, or those specified in airbyte. If the commit was made on a different branch, the information of who made it and when it was made is not stored. This PR is to solve that, to store the information of who and when the commit was made in a branch other than the main one (or those specified in airbyte)

**Review by thomas-gerber — APPROVED** [review]
@ted-faros confirmed that non-keys fields will be saved in CR
This is also consistent with https://github.com/faros-ai/airbyte-connectors/blob/main/destinations/airbyte-faros-destination/src/converters/github/commits.ts#L27

**Comment by thomas-gerber** [comment]
@tovbinm what should we do w.r.t. unsigned commits from @jaorr95 ?

**Pull request merged**